tsconfig.json

```diff
@@ -1,19 +1,28 @@
 {
     "compilerOptions": {
+        "target": "ESNext",
         "module": "ESNext",
         "moduleResolution": "Bundler",
         "types": ["node", "vitest/globals"],
-        "declaration": true,
+        "declaration": false,
         "outDir": "./dist",
         "esModuleInterop": true,
         "forceConsistentCasingInFileNames": true,
         "strict": true,
         "noImplicitAny": true,
         "skipLibCheck": true,
-        "emitDeclarationOnly": true,
         "jsx": "react-jsx",
-        "lib": ["ESNext"],
+        "lib": ["ESNext", "dom", "dom.iterable"],
```

Comment:
@acaldas a while ago I set this up to not have these dom libs in the main tsconfig and then add them in the /editors tsconfig. This seemed to work when I checked it in vs code, but when we run `tsc` directly it doesn't recognize the /editors tsconfig. Apparently the only way to make these nested tsconfigs work is to have a command that runs `tsc` twice, once for the main dir ignoring the /editors dir, and then again on just the /editors dir. This seems like a lot of added complexity. Do you think its worth it for the benefit of not allowing dom libs?

Reply:
It was just to make sure the reducers code can be run both on browser and NodeJS.
Node and dom are pretty compatible so I guess we don't need it. Don't see why a reducer would use browser-specific APIs.